addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -602,7 +602,7 @@ def _l10n_es_tbai_get_in_invoice_values_batuz(self, invoice):
         values['is_refund'] = invoice.move_type == 'in_refund'
         if values['is_refund']:
             values['credit_note_code'] = invoice.l10n_es_tbai_refund_reason
-            values['credit_note_invoice'] = invoice.reversed_entry_id
+            values['credit_note_invoices'] = invoice.reversed_entry_id | invoice.l10n_es_tbai_reversed_ids
```

Comment:
Do we need to keep `reversed_entry_id`?
Wouldn't it be easier to store everything in `l10n_es_tbai_reversed_ids` (the view would also be cleaner with only one field IMO)?

Reply:
@epictete The problem is that the reversed_entry_id is generic (and used by the credit note wizard, ...) while the other field is specific to TicketBai.